docs/pages/management/export-audit-events/fluentd.mdx

```diff
@@ -4,26 +4,56 @@ description: Forwarding events with Fluentd and Teleport event handler
 videoBanner: HAqxs4rBv2c
 ---
 
-In this guide, we will explain how to:
+Fluentd is an open-source data collector for a unified logging layer. In this
+guide, we will:
 
-- Set up Teleport's event handler.
+- Set up Teleport's Event Handler plugin.
 - Forward events with Fluentd.
 
+This guide also serves as an explanation for the Teleport Event Handler plugin,
+using Fluentd as the target service. We'll create a local Docker container as a
+destinatation for the Event Handler:
+
+{/*
+ The Graph below is made with this mermaidjs logic:
+ graph LR
+subgraph workstation [fa:fa-laptop Workstation]
+    style workstation fill:#C0C0C0,stroke: #000000
+    direction LR
+    c(teleport-event-handler)
+    style c fill: #a379ff,color: #ffffff
+    subgraph Docker
+        style Docker fill:#0db7ed,stroke: #000000
+        d(fluentd)
+    end
+    c-->d
+end
+e(Teleport)
+style e fill: #844cff,color: #ffffff
+e-->c
+ */}
+
+![Fluentd example diagram](../../../img/management/fluentd-demo-dia.png)
+
+You can follow the instructions below for a local proof-of-concept demo, or use any
+of the additional installation instructions to configure the Teleport Event Handler
+directly into your infrastructure.
```

Comment:
```suggestion
to integrate with your infrastructure.
```
nit: I thought "configure ... directly into" sounded slightly off

Reply:
Agreed. Copy edits FTW! You stare at something long enough and you don't see the forest for the trees.